package.json

```diff
@@ -66,23 +67,26 @@
     "all-contributors-cli": "^6.24.0",
     "auto": "^10.42.2",
     "esbuild": "^0.16.14",
+    "esbuild-plugin-copy": "^2.1.1",
     "esbuild-plugin-text-replace": "^1.2.0",
-    "eslint": "^8.31.0",
+    "eslint": "8.52.0",
     "eslint-config-airbnb-base": "^15.0.0",
     "eslint-plugin-import": "^2.26.0",
     "eslint-plugin-jest": "^27.2.0",
     "jest": "^29.3.1",
     "jest-environment-node": "^29.3.1",
     "ovsx": "^0.7.1",
-    "prettier": "^2.8.1",
+    "prettier": "^3.1.0",
+    "prettier-eslint": "^16.1.2",
     "ts-node": "^10.9.1",
     "typescript": "^4.9.5",
     "vsce": "^2.15.0"
   },
   "dependencies": {
     "find-up": "^6.3.0",
     "ignore": "^5.2.4",
-    "prettier-eslint": "^15.0.1"
+    "require-relative": "^0.8.7",
+    "synckit": "^0.8.6"
```

Comment:
It's marked as dependency here, do we really need to bundle it? Again, I'm lacking knowledge of VSCode, feel free to tell that I'm wrong here.

Reply:
@JounQin I tried without bundling it and it causes the extension to hang.

Reply:
Let me see if I can help.